packages/insomnia-app/app/ui/components/settings/plugins.js

```diff
@@ -267,7 +278,17 @@ class Plugins extends React.PureComponent<Props, State> {
             <button className="pull-right icon" onClick={this._handleClearError}>
               <i className="fa fa-times" />
             </button>
-            <div className="selectable force-pre-wrap">{error}</div>
+            <div className="selectable force-pre-wrap">
+              <b>{installPluginErrMsg}</b>
```

Comment:
Since we can remove this from state, it can simply be:

```suggestion
              <b>Failed to install "{npmPluginValue}"</b>
```